pkg/odo/cli/component/list.go

```diff
@@ -46,11 +47,19 @@ func (lo *ListOptions) Complete(name string, cmd *cobra.Command, args []string)
 
 // Validate validates the list parameters
 func (lo *ListOptions) Validate() (err error) {
+	configExists, err := odoconfig.LocalConfigExists("")
+	if err != nil {
+		return err
+	}
+	if !configExists {
+		return fmt.Errorf("the current directory does not represent an odo component")
```

Comment:
would be great to have give a hint to the developer related to what they should do next like -
```
the current directory .... component
use "odo create" to create a component in the directory # maybe a better phrasing
```

Reply:
Addressed, pushing shortly